api/SVGImageElement.json

```diff
@@ -257,10 +257,10 @@
               "version_added": null
             },
             "firefox": {
-              "version_added": null
+              "version_added": false
```

Comment:
This (https://developer.mozilla.org/docs/Web/API/SVGImageElement/href) is set to true for firefox and firefox_android in https://github.com/mdn/browser-compat-data/pull/3728.

This is confirmed by the source:
https://github.com/mozilla/gecko-dev/blob/master/dom/webidl/SVGImageElement.webidl

It looks like you're not applying `SVGImageElement implements SVGURIReference`?

Reply:
Note to self: fix the WebIDL parser to merge all of the partials and implements together.

Reply:
And mixins if you're not already doing that. (There are also partial interface mixins, so there's a certain order everything has to be done in.)

Reply:
I just went through and fixed the script so that it gets EVERY piece of an interface and merges them together when checking members, `implements` definitions included.  Already, it found numerous entries that were defined as `false` (either in here or just in general; will split the changes to those that weren't originally `null` into a separate PR).